app/config/locales/en.yml

```
@@ -1,5 +1,35 @@
 ---
 en:
+  activerecord:
+    errors:
+      models:
+        cbv_flow_invitation:
+          attributes:
+            agency_id_number:
+              blank: Enter a valid agency ID number.
+              invalid_format: Agency ID number must be 7 digits.
+            beacon_id:
+              blank: can't be blank
+              invalid_format: Your WELID must be 6 characters. It can only include letters and numbers.
+            case_number:
+              invalid_format: Client's case number must be 12 digits. Enter the case number in the ANGIE/sPOS format, i.e. 00012345678A.
+            client_id_number:
+              invalid_format: Enter the client's CIN in the correct format, XX00000X.
+            email_address:
+              blank: Enter the client's email address.
+              invalid_format: Enter an email address in the correct format, like name@example.com
+            first_name:
+              blank: Enter the client's first name.
+            last_name:
+              blank: Enter the client's last name.
+            middle_name:
+              invalid_format: must be letters only
+            snap_application_date:
+              invalid_date: is not a valid date
+              ma_future_date: Enter today's date or the date you contacted the client. This date must be today or in the past 30 days.
+              ma_too_old: Enter today's date or the date you contacted the client. This date must be today or in the past 30 days.
```

Enter today's date or the date you contacted the client. This date must be today or in the past year.

---

@makaelastephens Looks like after you and I did a review there were some changes that I didn't push. We actually got rid of `ma_too_old ` and `ma_future_date` and kept a single `ma_invalid_date` and `nyc_invalid_date`. 

```suggestion
            snap_application_date:
              invalid_date: is not a valid date
              ma_invalid_date: Enter today's date or the date you contacted the client. This date must be today or in the past year.
              nyc_invalid_date: SNAP application/recertification date must be today or in the past 30 days.
```